test/validation/org/apache/cassandra/bridges/ArchiveClusterLogs.java

```diff
@@ -95,4 +105,62 @@ private static void compressFiles(ZipOutputStream zos,String folderName,String b
             throw new RuntimeException(e);
         }
     }
+
+    public static void savetempDirectoryPath(File CASSANDRA_DIR, File ccmDir)
```

Comment:
Rename the second argument. If this is supposed to be re-usable for more than ccm, the method should reflect that.